src/components/ZclAttributeManager.vue

```diff
@@ -267,20 +267,23 @@ export default {
         })
       }
     },
-    checkForcedExternal(name) {
-      if (
-        this.forcedExternal.byName?.[this.selectedCluster.label]?.includes(name)
-      ) {
-        return true
-      } else {
-        return false
-      }
+    checkForcedExternal(name, selectedCluster) {
+      let forcedExternal
+      this.forcedExternal.map((option) => {
```

Comment:
Same as above. Did you mean .map here or typo?

Reply:
fixed